tests/brun/any-3.txt

```diff
@@ -1,3 +1,3 @@
-brun -c '(all (q 0) (q 0))'
```

Comment:
the typo "all" -> "any" here has been fixed in `develop`. You should rebase to pick up this fix.